codegen/weave-device-cpp/templates/common.macros

```diff
@@ -53,7 +53,7 @@ enum {{ enum.base_name }} {
               chain(get_struct_dependencies(obj)|reject('common')|map(attribute='parent'))|
               chain(get_struct_dependencies(obj)|select('common'))|
               reject('false')|unique|reject('standard')|reject('vendor')|
-              reject('equalto',obj)|sort -%}
+              reject('equalto',obj)|sort(attribute='full_name') -%}
```

Comment:
This seems unrelated to the Python 3 upgrade. Is it?

Reply:
It is VERY much related.  In Python2, comparison between unrelated objects would yield a stable result, in Python3, in the absence of a comparison operator, this throws an error.  In this case, the objects sorted were derived from a common ancestor, but defined no comparison.  When we were comparing Typespace and Trait, the comparison would throw an error.   I would note that the new code parallels the structure we've seen in other templates for codegen that have not been opensourced yet.